src/components/fellows-header.js

```diff
@@ -1,37 +1,12 @@
 var React = require(`react`);
+var StickyContainer  = require(`./sticky-container.js`);
 var Router = require(`react-router`);
 var Link = Router.Link;
 
 module.exports = React.createClass({
-  getInitialState: function() {
-    window.scrollTo(0, 0);
-    return {
-      sticky: false
-    };
-  },
-  onScroll: function() {
-    var sticky = window.scrollY >= 448;
-
-    if (sticky !== this.state.sticky) {
-      this.setState({
-        sticky: sticky
-      });
-    }
-  },
-  componentDidMount: function() {
-    document.addEventListener(`scroll`, this.onScroll);
-  },
-  componentWillUnmount: function() {
-    document.removeEventListener(`scroll`, this.onScroll);
-  },
   render: function() {
-    var className = `fellows-header`;
-
-    if (this.state.sticky) {
-      className += ` sticky`;
-    }
     return (
-      <div className={className}>
+      <StickyContainer className="fellows-header" stickyFrom={() => 448}>
```

Comment:
We set this up to be sticky at scroll point above 448, otherwise if the scroll is below 448 it becomes fixed.

Fixed means it scrolls with the page, sticky means it sticks to another element.